src/main.js

```diff
@@ -97,40 +75,45 @@ Object.assign(extension, {
       const link = this.getElementToActivate(results, true);
       browser.runtime.sendMessage({
         type: 'tabsCreate',
-        options: { url: link.href, active: true }
+        options: {
+          url: link.href,
+          active: true
+        }
       });
     });
     this.register(options.navigateNewTabBackgroundKey, () => {
       const link = this.getElementToActivate(results, true);
       browser.runtime.sendMessage({
         type: 'tabsCreate',
-        options: { url: link.href, active: false }
+        options: {
+          url: link.href,
+          active: false
+        }
       });
     });
-    this.register(options.navigateShowAll, () => this.changeTools('a'));
-    this.register(options.navigateShowHour, () => this.changeTools('h'));
-    this.register(options.navigateShowDay, () => this.changeTools('d'));
-    this.register(options.navigateShowWeek, () => this.changeTools('w'));
-    this.register(options.navigateShowMonth, () => this.changeTools('m'));
-    this.register(options.navigateShowYear, () => this.changeTools('y'));
-    this.register(options.toggleSort, () => this.changeTools(null));
+    this.register(options.navigateShowAll, () => searchEngine.changeTools('a'));
+    this.register(options.navigateShowHour, () => searchEngine.changeTools('h'));
+    this.register(options.navigateShowDay, () => searchEngine.changeTools('d'));
+    this.register(options.navigateShowWeek, () => searchEngine.changeTools('w'));
+    this.register(options.navigateShowMonth, () => searchEngine.changeTools('m'));
+    this.register(options.navigateShowYear, () => searchEngine.changeTools('y'));
+    this.register(options.toggleSort, () => searchEngine.changeTools(null));
   },
 
-  initCommonGoogleSearchNavigation() {
+  initCommonGoogleSearchNavigation(tabs) {
```

Comment:
Rename to `initCommonSearchEngineNavigation`?

Reply:
Good idea. Better to remove search engine specifics from this file. Fixed in bd2a9dd